src/main/java/frc/robot/subsystems/AlgaeArm/AlgaeArmIOCB.java

```diff
@@ -32,21 +32,21 @@
 
 public class AlgaeArmIOCB implements AlgaeArmIO {
 
-  private final SparkMax armMotor = new SparkMax(Constants.CompBotConstants.ALGAE_ARM_ID, MotorType.kBrushless); // placeholder                                                                                                                    // ID
-  private final RelativeEncoder encoder = armMotor.getEncoder();
+  protected final SparkMax armMotor = new SparkMax(Constants.CompBotConstants.ALGAE_ARM_ID, MotorType.kBrushless); // placeholder                                                                                                                    // ID
```

Comment:
What's the rationale behind having these variables be protected rather then private?